### ISSUE TITLE
Typocheck

### ISSUE THREAD

**Issue by lfuelling** [lfuelling/textie-web#7]
Wenn wir fertig sind, müssen wir die Seite nochmal auf Rechtschreibfehler überprüfen.

**Comment by YuleZ** [comment]
@lfuelling

**Comment by lfuelling** [comment]
Ja, muss noch gemacht werden, wenigstens sage ich nicht einfach wontfix :tongue:
